dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV DEBIAN_FRONTEND="noninteractive"
 
 RUN apt-get update -qq && apt-get install -y -qq m4 wget time gcc-multilib opam \
         libgtk2.0-dev libgtksourceview2.0-dev \
-        texlive-latex-extra texlive-fonts-recommended hevea \
+        texlive-latex-extra texlive-fonts-recommended texlive-science \
```

Comment:
What is -science for?

Reply:
This is used for `textgreek` in the `source-doc` target.